### DISCUSSION TITLE
LyraT 4.3 Audio Player from SD

### DISCUSSION THREAD

**Discussion by souza107** [pschatzmann/arduino-audio-tools#91] [Q&A] [answered]
I am struggling to get the LyraT 4.3 Audio board to play an mp3 from SD. I know it may be asking a lot but would it be possible for you to give an example if this? From the examples I am not able to use the sdfat library with this board. I have been able to get the board to recognize the SD using sd.h library and detect the headphones but I can get an mp3 to play. Any help would be greatly appreciated

**Comment by pschatzmann** [comment, marked as answer]
Strange, for me the LyraT was working. If I remember right all dip switches needed to be in the on position.
I used the following sketch for the test:
https://github.com/pschatzmann/arduino-audio-tools/tree/main/examples/examples-audiokit/player-sd-audiokit

I'll try to retest this tomorrow...
